vendor_skins/Evercast/libobs-opengl/gl-x11.c

```diff
@@ -0,0 +1,579 @@
+/******************************************************************************
+    Copyright (C) 2014 by Zachary Lund <admin@computerquip.com>
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+******************************************************************************/
+
+/* Version 2 of the GLX backend...
+ * Difference from version 1 is that we use XCB to help alleviate
+ * pains in a threaded environment that is prone to error.
+ * These errors must be readable and handled for the sake of,
+ * not only the users' sanity, but my own.
+ *
+ * With that said, we have more error checking capabilities...
+ * and not all of them are used to help simplify current code.
+ *
+ * TODO: Implement more complete error checking.
+ * NOTE: GLX loading functions are placed illogically
+ * 	for the sake of convenience.
+ */
+
+#include <X11/Xlib.h>
```

Comment:
LIIIIIINUUUUX!

Reply:
2020 is the year of EBS on the linux desktop